docs/api/Differential.md

```diff
@@ -51,116 +64,116 @@ Initializes a new Differential instance.
 
 | Name | Type | Description |
 | :------ | :------ | :------ |
-| `apiSecret` | `string` | The API Secret for your Differential cluster. Obtain this from [your Differential dashboard](https://admin.differential.dev/dashboard). |
+| `apiSecret` | `string` | The API Secret for your Differential cluster. You can obtain one from https://api.differential.dev/demo/token. |
 
 #### Returns
 
 [`Differential`](Differential.md)
 
 #### Defined in
 
-[src/Differential.ts:374](https://github.com/differentialHQ/differential/blob/b306aab/ts-core/src/Differential.ts#L374)
+[src/Differential.ts:398](https://github.com/differentialHQ/differential/blob/44e4229/ts-core/src/Differential.ts#L398)
 
 ## Methods
 
-### background
+### client
 
-▸ **background**\<`T`, `U`\>(`fn`, `...args`): `Promise`\<\{ `id`: `string`  }\>
+▸ **client**\<`T`\>(`service`): `ServiceClient`\<`T`\>
 
-Calls a function on a registered service, while ensuring the type safety of the function call through generics.
-Returns the job id of the function call, and doesn't wait for the function to complete.
+Provides a type safe client for performing calls to a registered service.
+Waits for the function to complete before returning, and returns the result of the function call.
 
 #### Type parameters
 
 | Name | Type |
 | :------ | :------ |
-| `T` | extends `RegisteredService` |
-| `U` | extends `string` \| `number` \| `symbol` |
+| `T` | extends `RegisteredService`\<`any`\> |
 
 #### Parameters
 
-| Name | Type | Description |
-| :------ | :------ | :------ |
-| `fn` | `U` | The function name to call. |
-| `...args` | `Parameters`\<`T`[``"definition"``][``"functions"``][`U`]\> | The arguments to pass to the function. |
+| Name | Type |
+| :------ | :------ |
+| `service` | `T`[``"definition"``][``"name"``] |
 
 #### Returns
 
-`Promise`\<\{ `id`: `string`  }\>
+`ServiceClient`\<`T`\>
 
-The job id of the function call.
+ServiceClient<T>
 
 **`Example`**
 
 ```ts
 import { d } from "./differential";
+import type { helloService } from "./hello-service";
 
-const result = await d.background<typeof helloService, "hello">("hello", "world");
+const client = d.buildClient<typeof helloService>();
```

Comment:
Might need to re-generate the docs? (`buildClient` -> `client`)

Reply:
Thanks. Missed a spot.